lib/locations/location.rb

```diff
@@ -1,3 +1,3 @@
 module Locations
-  Location = Struct.new(:id, :name, :address, :phone, :hours, :lat_lng)
+  Location = Struct.new(:id, :name, :address, :booking_centre, :phone, :hours, :lat_lng)
```

Comment:
Given booking centre is just another location, wouldn't `booking_location` be more revealing?